aplib.net-demo/Assets/Scripts/Player/Inventory.cs

```diff
@@ -0,0 +1,115 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Inventory : MonoBehaviour
+{
+    //TODO Add icon logic to Item class
+    //TODO Add hotbar Object which contains 4 subjobjects. They are linked to the inventory class and will display the right icons depending on inventory contents.
+    //TODO Add controls for using items to InputManager(?) class
+    //TODO Add controls for switching which item you're using to InputManager(?) class
+    //TODO Add logic for items with a duration and effect
```

Comment:
This is too many TODO's, they should only be used to be able to check quickly from the jira ticket where something can be implemented but this is too much, if they are actual features add to the backlog channel on discord.